src/engine/SCons/Platform/__init__.py

```diff
@@ -100,21 +101,15 @@ def platform_module(name = platform_default()):
             eval(full_name)
         else:
             try:
-                file, path, desc = imp.find_module(name,
-                                        sys.modules['SCons.Platform'].__path__)
-                try:
-                    mod = imp.load_module(full_name, file, path, desc)
-                finally:
-                    if file:
-                        file.close()
+                mod = importlib.import_module(full_name)
```

Comment:
I think we can do a relative import here as well, using .name instead of full_name

Reply:
done as agreed above.